monkestation/code/controllers/subsystem/plexora.dm

```diff
@@ -0,0 +1,370 @@
+/**
+ * # Plexora Subsystem
+ *
+ * This subsystem is for the Plexora Discord bot bridge.
+ *
+ * The repo for this can be found at https://github.com/monkestation/plexora
+ *
+ * The distinction between Plexora (the bot) and Plexora (the subsystem)
+ * will be plexora (the bot) and SSplexora (the subsystem)
+ *
+ * NOTES:
+ * * SSplexora makes heavy use of topics, and rust_g HTTP requests
+ * * Lets hope to god plexora is configured properly and DOESNT CRASh,
+ *   -because i seriously do not want to put error catchers in
+ *   -EVERY FUNCTION THAT MAKES AN HTTP REQUEST
+ */
+
+#define AUTH_HEADER ("Basic " + CONFIG_GET(string/comms_key))
+
+#define HTTP_DEFAULT_HEADERS(...) \
+	var/list/headers = list(); \
+	headers["Content-Type"] = "application/json"; \
+	headers["Authorization"] = AUTH_HEADER;
+
+SUBSYSTEM_DEF(plexora)
+	name = "Plexora"
+	wait = 30 SECONDS
+	init_order = INIT_ORDER_PLEXORA
+	priority = FIRE_PRIORITY_PLEXORA
+	runlevels = ALL
+
+	var/version_increment_counter = 1
+	var/configuration_path = "config/plexora.json"
+	var/plexora_is_alive = FALSE
+	var/http_root = ""
+	var/http_port = 0
+	var/enabled = TRUE
+
+/datum/controller/subsystem/plexora/Initialize()
+	. = ..()
+
+	if (!rustg_file_exists(configuration_path))
```

Comment:
```suggestion
/datum/controller/subsystem/plexora/Initialize()
	if (!rustg_file_exists(configuration_path))
```
no need to call parent for subsystem initialize, as parent is literally only `return SS_INIT_NONE`